allocate.go

```diff
@@ -50,7 +50,7 @@ func setupExecAllocator(opts ...ExecAllocatorOption) *ExecAllocator {
 
 // DefaultExecAllocatorOptions are the ExecAllocator options used by NewContext
 // if the given parent context doesn't have an allocator set up. Do not modify
-// this global; instead, use NewExecAllocator. See ExampleExecAllocator.
+// this global; instead, use NewExecAllocator. See the example for [ExecAllocator].
```

Comment:
It is not clear which example is for `ExecAllocator`. I can not find a way to link to the func `ExampleExecAllocator`. If we want to do anything here, maybe the following change is better:

```diff
- // this global; instead, use NewExecAllocator. See ExampleExecAllocator.
+ // this global; instead, use NewExecAllocator. See [ExampleExecAllocator].
+ //
+ // [ExampleExecAllocator]: https://pkg.go.dev/github.com/chromedp/chromedp#example-ExecAllocator
```

Reply:
Changed